client/jetpack-cloud/sections/partner-portal/license-details/bundle-details.tsx

```diff
@@ -7,28 +9,41 @@ interface Props {
 }
 
 export default function BundleDetails( { parentLicenseId }: Props ) {
-	const { data } = useBundleLicensesQuery( parentLicenseId );
+	const translate = useTranslate();
+	const { licenses, total, loadMore, fetching } = useBundleLicensesQuery( parentLicenseId );
```

Comment:
Can we rename `fetching` to `isFetching`?

Reply:
I have removed this in favor of `isLoading`.